src/main/java/com/checkmarx/intellij/tool/window/results/tree/nodes/ResultNode.java

```diff
@@ -399,6 +447,18 @@ private void handleGeneralRemediationClick(JLabel remediation, Result result) {
         }));
     }
 
+    /**
+     * Create a section title in the panel
+     *
+     * @param scaBody - body panel
+     * @param resource - message recourse
+     */
+    private void addSectionTitle(JPanel scaBody, Resource resource) {
+        JLabel sectionTitle = new JLabel(String.format("<html><b>%s</b></html>", boldLabel(Bundle.message(resource)).getText()));
```

Comment:
Let's add a format here, similar to HTML_FONT_BLUE_FORMAT

Reply:
Fixed